metric_source/local/local.go

```diff
@@ -18,23 +19,26 @@ import (
 // Local is implementation of MetricSource interface, which implements fetch metrics method from moira database installation
 type Local struct {
 	dataBase moira.Database
+	logger   moira.Logger
 }
 
 // Create configures local metric source
-func Create(dataBase moira.Database) metricSource.MetricSource {
+func Create(dataBase moira.Database, logger moira.Logger) metricSource.MetricSource {
 	// configure carbon-api functions
 	rewrite.New(make(map[string]string))
 	functions.New(make(map[string]string))
 
 	return &Local{
 		dataBase: dataBase,
+		logger:   logger,
 	}
 }
 
 // Fetch is analogue of evaluateTarget method in graphite-web, that gets target metrics value from DB and Evaluate it using carbon-api eval package
 func (local *Local) Fetch(target string, from int64, until int64, allowRealTimeAlerting bool) (metricSource.FetchResult, error) {
 	// Don't fetch intervals larger than metrics TTL to prevent OOM errors
 	// See https://github.com/moira-alert/moira/pull/519
+	startFetch := time.Now()
```

Comment:
Lets move time measure and logging to wrapper function, it is noisy code in place of business logic. 
Also this line separates comment and next line (warning about OOM)